### ISSUE TITLE
Understand the notes patter and make contribute.md

### ISSUE THREAD

**Issue by shreeshailaya** [shreeshailaya/C-DAC-Notes#61]
Our project is in under construction so we need to create contribute.md for hacktoberfest and for others contributors

**Comment by nikitadhona** [comment]
Can i work upon this issue and create contributing.md for hacktoberfest?

**Comment by shreeshailaya** [comment]
yes go ahead but before that make sure you know the structure of this repository, please refer [Core Java Notes](https://github.com/shreeshailaya/C-DAC-Notes/tree/main/Core%20Java) like core java we need to create notes of all subjects with their assignments.

**Comment by shreeshailaya** [comment]
@nikitadhona if you need any help with doing this feel free to ask.

**Comment by shreeshailaya** [comment]
I'm closing this issue and reopening with same title..
